packages/browserslist-config/CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# 2.0.0
+
+-   Transfer to @wordpress npm orginization
```

Comment:
`orginization` -> `organization`.

Reply:
Thanks @atimmer 👍 Fixed in https://github.com/WordPress/packages/pull/69/commits/398fae0f0456da6207ffff0849c508b2382135dc